pkg/pullexecutor/pullexecutor.go

```diff
@@ -119,7 +123,11 @@ func (m *PullExecutor) StartPulling(o *PullOptions) error {
 					m.asyncErrs[o.NamedRef] = fmt.Errorf("unable to pull image %q: %s", o.Image, err)
 					return
 				}
-				metrics.ImagePullTime.WithLabelValues(metrics.Async).Observe(time.Since(startTime).Seconds())
+				elapsed := time.Since(startTime)
+				metrics.ImagePullTime.WithLabelValues(metrics.Async).Observe(elapsed.Seconds())
+				valuesLogger.Info(fmt.Sprintf("pulling %q took %s", o.Image, elapsed))
+				size, _ := puller.ImageSize(o.Context)
+				valuesLogger.Info(fmt.Sprintf("image size: %d MB", size))
```

Comment:
I'm not sure what units to report here. The go package [docs](https://pkg.go.dev/k8s.io/cri-api/pkg/apis/runtime/v1alpha2#ImageFsInfoResponse.Size) don't specify, and when I compare the output with what I see in `ctr` it's different: 
![image](https://github.com/warm-metal/csi-driver-image/assets/22019221/cf75e0b5-e123-49a7-96d8-c0434ba7c8a4)

Reply:
Let's try using puller.ImageSize() to check what size is reported.

Also let's try to verify the outcome by using different `volID` (for two different images) in the tests.